example/pubspec.yaml

```diff
@@ -27,6 +27,7 @@ dependencies:
   workmanager: ^0.5.2
   firebase_messaging: ^14.9.2
   firebase_core: ^2.31.0
+  open_file: ^3.5.4
```

Comment:
I remember open-file didn't work correctly for our usecase, could be removed?

Reply:
keeping it on and lets try to make it work in upcoming release, its in example app so no changes in SDK.